packages/clerk-js/src/utils/pageLifecycle.ts

```diff
@@ -16,22 +16,22 @@ const noop = () => {
  */
 export const createPageLifecycle = () => {
   if (!inBrowser()) {
-    return { isUnloading: noop, onPageVisible: noop };
+    return { onPageFocus: noop };
   }
 
   const callbackQueue: Record<string, Array<() => void>> = {
-    'visibilitychange:visible': [],
+    focus: [],
   };
 
-  document.addEventListener('visibilitychange', () => {
+  window.addEventListener('focus', () => {
```

Comment:
`focus` seems to be a better indicator of activity, and avoids an edge case with visibility where two windows are open side-by-side, both being visible.